libblockchain/BlockChainImp.cpp

```diff
@@ -879,14 +879,12 @@ CommitResult BlockChainImp::commitBlock(Block& block, std::shared_ptr<ExecutiveC
             writeTotalTransactionCount(block, context);
             writeTxToBlock(block, context);
             context->dbCommit(block);
+            {
```

Comment:
https://github.com/FISCO-BCOS/FISCO-BCOS/blob/869585e2897db45379e290cb294ffe803604cddf/libblockchain/BlockChainImp.cpp#L838-L849

I really think this `isBlockShouldCommit` is unnecessary😔😔😔😔